Source/Tools/LinuxEmulation/LinuxSyscalls/Syscalls.h

```diff
@@ -255,7 +256,16 @@ class SyscallHandler : public FEXCore::HLE::SyscallHandler, FEXCore::HLE::Source
 protected:
   SyscallHandler(FEXCore::Context::Context* _CTX, FEX::HLE::SignalDelegator* _SignalDelegation);
 
-  fextl::vector<SyscallFunctionDefinition> Definitions {};
+  static auto constexpr cvt = [](auto in) {
```

Comment:
This funky lambda is still visible in the header. I'm guessing it's here to avoid duplicating it, but if it stays in the visible interface, it should be converted to a template member function and it should be given a more suitable name + short docstring.

Reply:
Ah, misinterpreted the comment originally. Will fix.

Reply:
Just removed the lambda since all it is doing is a reinterpret_cast